src/content/docs/developers/widget-tool.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Widget tool
+---
+
+import { LinkButton } from '@astrojs/starlight/components';
+import { LinkOut } from "@interledger/docs-design-system";
+import Connect from '/src/partials/tools-connect-wallet.mdx'
+import Change from '/src/partials/tools-change-wallet.mdx'
+
+Add a <LinkOut href="https://webmonetization.org/tools/widget">payment widget</LinkOut> to your site so visitors can send you a one-time payment. The widget appears to all visitors, even those with an active Web Monetization extension. 
+
+### Before you begin
+
+* Your visitor must have a wallet address with a [compatible wallet](/wallets) to send a payment using the widget.
+* If your site is monetized with multiple wallet addresses, only the address connected to the widget will receive one-time payments. You can [change the address](#connect-widget-to-a-new-wallet-address) connected to the widget at any time.
+
+## Create a new widget
+
+### Connect your wallet
+
+<Connect />
+
+### Change the widget's contents
+
+1. Optionally change the label of a configuration by selecting the pencil icon and typing over the default.<br /><img src="/img/docs/pubtools/tools-preset-labels.png" alt="Screen with three tabs, all with default labels, and a pencil icon appearing next to the first label" style="max-width: 60%" class='img-outline' />
+2. Expand the *Content* drop-down menu. Enter a title for the widget or choose from the suggested options.<br /><img src="/img/docs/pubtools/widget-title.png" alt="Panel showing suggested titles and a field to enter a custom title" style="max-width: 60%" class='img-outline' />
+3. Enter the message to show to your visitors or use the default text. We suggest changing the message so that it's tailored specifically to your audience.<br /><img src="/img/docs/pubtools/widget-message.png" alt="Panel with a checkbox to mark the message as visible and a blank field to enter a custom message" style="max-width: 60%" class='img-outline' />
+4. Review your changes in the preview panel.<br /><img src="/img/docs/pubtools/widget-content-preview.png" alt="Preview panel showing how the widget will appear with a custom title and message" style="max-width: 60%" class='img-outline' />
+5. Click **Done** to save your changes.
```

Comment:
"Done" just collapses the section/accordion. Doesn't _save_ really save. It's only a UI affordance to keep view clear.